packages/tool-browser/src/readiness.ts

```
@@ -48,6 +48,7 @@ export async function probeBrowserCapability(options: {
         abortSignal,
       });
       await assertCdpHttpReady(`${endpointFor(sandbox, cdpPort)}/json/version`, abortSignal);
+      let lastProbeFailure: unknown;
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)